react-version/src/components/game/BoardButtons.js

```diff
@@ -0,0 +1,1221 @@
+import React, { Fragment } from "react";
+
+const BoardButtons = (props) => {
+	return (
+		<Fragment>
+			<div className="row16-container" id="row16" title="16" data-value="16">
+				<button
+					type="button"
+					className="btn1 btn-line16 cave"
+					id="165"
+					title="fjord"
+					value="1"
+				></button>
```

Comment:
Those buttons should be rendered inside the loop. I hope you will fix it soon :)